Documentation/building/windows-instructions.md

```diff
@@ -102,6 +102,11 @@ Once you've built the source code for netfx from the root (`build.cmd -framework
 For advanced debugging using WinDBG see [Debugging CoreFX on Windows](https://github.com/dotnet/corefx/blob/master/Documentation/debugging/windows-instructions.md)
 
 ### Notes
+* At any given time, the corefx repo might be configured to use a more recent compiler than
+the one used by the most recent Visual Studio IDE release.  This means the corefx codebase might
+be using language features that are not understood by the IDE, which might result in errors that
+show up as red squiggles while writing code.  Such errors should, however, not affect the actual compilation.
```

Comment:
Point to the VSIX that supports the new language features?

Reply:
@VSadov, is there a stable link from which a dev can always get the latest vsix? I'd prefer not to include a vsix link we need to continually keep up to date.

Reply:
I've added a link to the CoreFxLab "how to build" section. 
Will add a link to CoreFx too.

Note: the feature has been merged to master last week. Once we get drops of VS with that, the requirement will be basically "recent version of VS"